cli/src/cmd_docs.rs

```diff
@@ -61,9 +61,23 @@ fn to_json(cmd: &Command) -> JsonDoc {
     }
 }
 
+#[derive(Serialize, Debug, PartialEq, Eq)]
+pub struct OutputJson {
+    version: String,
+
+    #[serde(flatten)]
+    commands: JsonDoc,
+}
+
 impl CmdDocs {
     pub async fn run(&self, app: &Command) -> Result<()> {
-        let pretty_json = serde_json::to_string_pretty(&to_json(app))?;
+        const CLI_VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
While this works in the repo under `cargo run`, it won't work for a compiled binary since it's just reading from the `CARGO_PKG_VERSION` environment variable. Take a look at `cmd_version.rs` to see how to do it.